pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/JSONParser.groovy

```diff
@@ -103,8 +103,8 @@ class JSONParser {
                 case 'tools':
                     pipelineDef.tools = parseTools(pipelineJson.getJSONArray("tools"))
                     break
-                case 'properties':
-                    pipelineDef.properties = parseJobProperties(pipelineJson.getJSONObject("properties"))
+                case 'options':
```

Comment:
Oh, wait, no, `JSONParser` doesn't bother with those warnings, since nothing's actually using the JSON yet. =) Also, the error will show up at schema validation time, before we get to `JSONParser`.